core/cas-server-core-util-api/src/main/java/org/apereo/cas/util/cipher/BaseBinaryCipherExecutor.java

```diff
@@ -46,7 +46,7 @@
 
     private final SecretKeySpec encryptionKey;
 
-    private final Cipher aesCipher;
+    private String cipherAlgorithm = "AES";
```

Comment:
Seems like this should actually be a constant? final field?

Reply:
My mistake, I have changed that to `final` field